fab/inventory/staging-va

```diff
@@ -43,5 +43,7 @@ hqdb0-staging.internal-va.commcarehq.org
 hqdb0-staging.internal-va.commcarehq.org
 
 [riakcs]
+hqdjango4-staging.internal.commcarehq.org
```

Comment:
This will change the "control node" (which is the first node in this list) for the cluster, and will distribute authentication credentials from that machine instead. If this has never been run against the new cluster then it's fine. If it has, you might want to keep `hqdjango0-staging.internal-va.commcarehq.org` at the top.